RecyclerView/app/src/main/java/net/boondockradio/recyclerview/MainActivity.java

```diff
@@ -0,0 +1,85 @@
+package net.boondockradio.recyclerview;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.ArrayList;
+
+public class MainActivity extends AppCompatActivity {
+
+    private ArrayList<String> states;
+    private RecyclerView recyclerView;
+    private RecyclerView.Adapter adapter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        createDummyData();
+
+        recyclerView = (RecyclerView) findViewById(R.id.recycler_main_activity);
+        recyclerView.setHasFixedSize(true);
+        recyclerView.setLayoutManager(new LinearLayoutManager(this));
+
+        adapter = new StatesAdapter(states);
+        recyclerView.setAdapter(adapter);
```

Comment:
Well technically in here if we were to write we can write this way as well and I don't feel this is uncommon at all:

```java
recyclerView.setAdapter(new StatesAdapter(states));
```